Support missing expects method while using MigrateAtToConsecutiveExpectations

# Bug Report

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.10.6              |
| Installed as   | prefixed Rector                           |

Using the `MigrateAtToConsecutiveExpectationsRector` rule. If a mock was configured without calling the `expects()` method, an error would be threw.

Omitting `expects()` while configuring a mock is valid syntax.

# Minimal PHP Code Causing Issue
```php
<?php

namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;

class Foo
{
    public function someMethod()
    {
    }
}
final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
{
    protected $mock;

    protected function setUp(): void
    {
        parent::setUp();

        $this->mock = $this->createMock(Foo::class);
    }

    public function test(): void
    {
        $this->mock
            ->method('someMethod')
            ->willReturn(1);

        $mock = $this->createMock(Foo::class);

        $mock
            ->method('someOtherMethod')
            ->willReturn(1);
    }
}
```

```php
return static function (ContainerConfigurator $containerConfigurator): void {
    $services = $containerConfigurator->services();

    $services->set(\Rector\PHPUnit\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector::class);
};
```

# Actual Behaviour
```
 [ERROR] Could not process "..." file,
         due to:                                                                                                        
         "Argument 1 passed to Rector\PHPUnit\NodeAnalyzer\ExpectationAnalyzer::isValidExpectsCall() must be an instance
         of PhpParser\Node\Expr\MethodCall, instance of PhpParser\Node\Expr\PropertyFetch given, called in              
         vendor/rector/rector-prefixed/vendor/rector/rector-phpunit/src/NodeAnalyzer/ExpectationAnalyzer.php:49".
```

--- a/src/NodeAnalyzer/ExpectationAnalyzer.php
+++ b/src/NodeAnalyzer/ExpectationAnalyzer.php
@@ -148,10 +148,11 @@ final class ExpectationAnalyzer
 
     private function getExpects(Expr $expr, MethodCall $methodCall): Expr
     {
-        if (! $this->testsNodeAnalyzer->isInPHPUnitMethodCallName($expr, 'with')) {
-            return $methodCall->var;
-        }
         if (! $expr instanceof MethodCall) {
+            return $methodCall;
+        }
+
+        if (! $this->testsNodeAnalyzer->isInPHPUnitMethodCallName($expr, 'with')) {
             return $methodCall->var;
         }
         return $expr->var;

--- a/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
+++ b/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
@@ -10,58 +10,8 @@ class Foo
 }
 final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
 {
-    protected $mock;
-
-    protected function setUp(): void
-    {
-        parent::setUp();
-
-        $this->mock = $this->createMock(Foo::class);
-    }
-
     public function test(): void
     {
-        $this->mock
-            ->method('someMethod')
-            ->willReturn(1);
-
-        $mock = $this->createMock(Foo::class);
-
-        $mock
-            ->method('someOtherMethod')
-            ->willReturn(1);
-    }
-}
-
-?>
------
-<?php
-
-namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;
-
-class Foo
-{
-    public function someMethod()
-    {
-    }
-}
-final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
-{
-    protected $mock;
-
-    protected function setUp(): void
-    {
-        parent::setUp();
-
-        $this->mock = $this->createMock(Foo::class);
-    }
-
-    public function test(): void
-    {
-        $this->mock
-            ->method('someMethod')
-            ->willReturn(1);
-
         $mock = $this->createMock(Foo::class);
 
         $mock

--- a/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
+++ b/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
@@ -1,0 +1,73 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;
+
+class Foo
+{
+    public function someMethod()
+    {
+    }
+}
+final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
+{
+    protected $mock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->mock = $this->createMock(Foo::class);
+    }
+
+    public function test(): void
+    {
+        $this->mock
+            ->method('someMethod')
+            ->willReturn(1);
+
+        $mock = $this->createMock(Foo::class);
+
+        $mock
+            ->method('someOtherMethod')
+            ->willReturn(1);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\MigrateAtToConsecutiveExpectationsRector\Fixture;
+
+class Foo
+{
+    public function someMethod()
+    {
+    }
+}
+final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
+{
+    protected $mock;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->mock = $this->createMock(Foo::class);
+    }
+
+    public function test(): void
+    {
+        $this->mock
+            ->method('someMethod')
+            ->willReturn(1);
+
+        $mock = $this->createMock(Foo::class);
+
+        $mock
+            ->method('someOtherMethod')
+            ->willReturn(1);
+    }
+}
+
+?>

--- a/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
+++ b/tests/Rector/ClassMethod/MigrateAtToConsecutiveExpectationsRector/Fixture/skip_missing_expects.php.inc
@@ -15,7 +15,7 @@ final class SkipMissingExpects extends \PHPUnit\Framework\TestCase
         $mock = $this->createMock(Foo::class);
 
         $mock
-            ->method('someOtherMethod')
+            ->method('someMethod')
             ->willReturn(1);
     }
 }

